.gitignore

```
@@ -2,6 +2,9 @@
 # Files already tracked by Git are not affected.
 # See: https://git-scm.com/docs/gitignore
 
+## Project Specific ##
+publish_egress_ip.zip
```

In concert with the previous suggestion.

```suggestion
lambda_build.zip
```